app/assets/javascripts/ingredient_families.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
This is the reason why I don't like automatic models generation - it spawns lots of unnecessary files we don't use that sit there for no reason. Can you delete these?

Reply:
Ok. Done.